catalog-rest-service/src/main/java/org/openmetadata/catalog/exception/CatalogGenericExceptionMapper.java

```diff
@@ -39,6 +39,7 @@
 public class CatalogGenericExceptionMapper implements ExceptionMapper<Throwable> {
   @Override
   public Response toResponse(Throwable ex) {
+    ex.printStackTrace();
```

Comment:
we need to remove this

Reply:
I need to get my debug logs going with my unit tests :)